keylime/payloads/openshift-workers/local_action_remove_approval.py

```diff
@@ -2,6 +2,10 @@
 import asyncio
 
 async def execute(revocation):
+    logging.info('REVOCATION: ' + pp.pformat(revocation))
```

Comment:
It looks like we're missing an `import pprint` for this `pp` instance.

Reply:
Good catch, I didn't fully remove all my debugging statements. Will have a follow up commit to do this